AzSentinel/Public/Get-AzSentinelAlertRule.ps1

```diff
@@ -39,7 +41,12 @@ function Get-AzSentinelAlertRule {
         [Parameter(Mandatory = $false,
             ValueFromPipeline)]
         [ValidateNotNullOrEmpty()]
-        [Kind[]]$Kind
+        [Kind[]]$Kind,
+
+        [Parameter(Mandatory = $false,
+            ValueFromPipeline)]
+        [ValidateNotNullOrEmpty()]
+        [DateTime]$LastModified
```

Comment:
Can you please add an example line with the right time format etc for users

Reply:
Have added a second example with LastModified